src/objects/zcl_abapgit_object_tran.clas.abap

```diff
@@ -838,8 +841,6 @@ CLASS zcl_abapgit_object_tran IMPLEMENTATION.
       iv_tcode      = 'SE93'
       it_bdcdata    = lt_bdcdata ).
 
-    rv_exit = abap_true.
```

Comment:
huh?

Reply:
good catch. it did a PR from a system that wasn't up-to-date